pages/checkout.vue

```diff
@@ -123,45 +120,45 @@ import {
   SfButton,
   SfPersonalDetails,
   SfShipping,
-  SfPayment,
   SfConfirmOrder,
   SfOrderReview,
   SfCheckbox,
   SfInput,
 } from "@storefront-ui/vue"
 import { useAsync } from "@nuxtjs/composition-api"
-import { useCheckout, useCart, useUiState } from "@/composables"
-import { useNuxtApp } from "#app"
+import { useCheckout, useCart, useUiState, usePaymentMethods, useUser } from "@/composables"
+import { computed, ref, useNuxtApp } from "#app"
+import { buildPaymentMethodInput, defaultPaymentDetails } from "@/composables/helpers"
 
 export default {
   name: "Checkout",
   components: {
     SfSteps,
     SfPersonalDetails,
     SfShipping,
-    SfPayment,
     SfConfirmOrder,
     SfOrderReview,
     SfButton,
     SfCheckbox,
     SfInput,
   },
-  setup() {
+  setup(_prop, context) {
```

Comment:
if we are not using props, we can use only an underscore

Reply:
Done